src/node.cc

```diff
@@ -3261,6 +3258,9 @@ void LoadEnvironment(Environment* env) {
   // are not safe to ignore.
   try_catch.SetVerbose(false);
 
+  // Compile, execute the lib/internal/bootstrap_node.js file. (Which was
+  // included as static C string in node_natives.h.
+  // 'internal_bootstrap_node_native is the string containing that source code.)
   Local<String> script_name = FIXED_ONE_BYTE_STRING(env->isolate(), "node.js");
```

Comment:
Perhaps this script name should also be updated too since it has been named `bootstrap_node.js` for some time now?